src/devices.jl

```diff
@@ -46,7 +53,7 @@ end
 function totalmem(dev::CuDevice)
     mem_ref = Ref{Csize_t}()
     @apicall(:cuDeviceTotalMem, (Ptr{Csize_t}, CuDevice_t), mem_ref, dev)
-    return mem_ref[]
+    return convert(Int, mem_ref[])
```

Comment:
Was hoping to make things more user friendly, it's a lot easier to read  as an Int  as opposed toa UInt c.f.
```julia
julia> totalmem(CuDevice(0))
0x0000000080000000

#vs  
julia> totalmem(CuDevice(0))
2147483648
```